### DISCUSSION TITLE
Calibre 4.23 and Versions 6.80 and 6.81 of DeDRM

### DISCUSSION THREAD

**Discussion by thetechnodino** [apprenticeharper/DeDRM_tools#1455] [Q&A]
Probably operator error but When I install version 6.80 or 6.81, in Calibre 4.23, I receive an error:

All files downloaded and attempted install on 12/20/2020.

> InvalidPlugin: The plugin in u'C:\\Users\\John\\Downloads\\DeDRM_tools_6.8.0.zip' is invalid. It does not contain a top-level __init__.py file

The entire error message:

> calibre, version 4.23.0
ERROR: Unhandled exception: <b>InvalidPlugin</b>:The plugin in u'C:\\Users\\John\\Downloads\\DeDRM_tools_6.8.0.zip' is invalid. It does not contain a top-level __init__.py file

calibre 4.23 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.19041-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 2.7.16+
Windows: ('10', '10.0.19041', 'SP0', u'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: Gather KFX-ZIP (from KFX Input) (1, 36, 0) && Package KFX (from KFX Input) (1, 36, 0) && KFX metadata reader (from KFX Input) (1, 36, 0) && KFX Input (1, 36, 0)
Traceback (most recent call last):
  File "site-packages\calibre\gui2\preferences\plugins.py", line 317, in add_plugin
  File "site-packages\calibre\customize\ui.py", line 472, in add_plugin
  File "site-packages\calibre\customize\ui.py", line 61, in load_plugin
  File "site-packages\calibre\customize\zipplugin.py", line 194, in load
  File "site-packages\calibre\customize\zipplugin.py", line 292, in _locate_code
InvalidPlugin: The plugin in u'C:\\Users\\John\\Downloads\\DeDRM_tools_6.8.0.zip' is invalid. It does not contain a top-level __init__.py file

Any assistance appreciated.
Thanks
John

**Comment by ElleKayEm** [comment]
Unzip DeDRM_tools_6.8.0.zip and load DeDRM_Plugin.zip.
